src/credentials/CHIPCertToX509.cpp

```diff
@@ -543,6 +545,41 @@ static CHIP_ERROR DecodeConvertTBSCert(TLVReader & reader, ASN1Writer & writer,
     return err;
 }
 
+/**
+ * Variant of DecodeConvertTBSCert that handles reading a compact-pdc-identity
+ * where only the subject public key is actually encoded. All other values are
+ * populated / written as the well-known values mandated by the specification.
+ *
+ * Note: The reader must be positioned on the EllipticCurvePublicKey element.
+ */
+static CHIP_ERROR DecodeConvertTBSCertCompactIdentity(TLVReader & reader, ASN1Writer & writer, ChipCertificateData & certData)
```

Comment:
Suggest properly naming the out arguments (`outWriter`, `outCertData`) or such.

Reply:
I don't mind that as a general convention, but I didn't notice a single method in ChipCert.h following that pattern, so I've gone with fitting in with the existing code.